src/ellipsoid.jl

```diff
@@ -26,6 +26,8 @@ Base.hash(b::Ellipsoid, h::UInt) = hash(b.c, hash(b.ri2, hash(b.p, hash(b.data,
 Base.in(x::SVector{N}, b::Ellipsoid{N}) where {N} = sum((b.p * (x - b.c)).^2 .* b.ri2) ≤ 1.0
 
 function surfpt_nearby(x::SVector{N}, b::Ellipsoid{N}) where {N}
+    x == b.c && ((_m,i) = findmax(b.ri2); nout = b.p[i,:]; return (b.c + nout/√b.ri2[i], nout))  # assume b.p is orthogonal
```

Comment:
This is too many statements for one line; would be clearer to write out as `if x == b.c ...`.